converters/fromZigbee.js

```diff
@@ -3093,6 +3093,7 @@ const converters = {
         cluster: 'ssIasAce',
         type: 'commandArm',
         convert: (model, msg, publish, options, meta) => {
+            if (hasAlreadyProcessedMessage(msg, msg.data[1])) return;
```

Comment:
I've added this, can you check if that fixes the duplicate messages.

Reply:
i added the line, but this not fix
https://pastebin.com/g0q1EzJ6
thanks

Reply:
Can you provide the herdsman debug logging of this?

To enable herdsman debug logging, see https://www.zigbee2mqtt.io/information/debug.html#zigbee-herdsman-debug-logging

Reply:
i added line to my docker-compose

`    environment:
      - TZ=Europe/Paris
      - DEBUG=zigbee-herdsman*
`
here the logs : 
https://pastebin.com/2MpYy9nr

Reply:
Logging is only logged to stdout, not log files. Use docker logs to get them.

Reply:
here the good one :)  https://pastebin.com/gJUYK9B4

Reply:
here the good one :) https://pastebin.com/gJUYK9B4

Reply:
Looks the remote is buggy, you see that it sends out multiple zigbee message with separate sequence transaction numbers which means separate events).

Reply:
yes i think so :/

Reply:
Just received two of this type. I think it is not a bug since I see the light on the remote turn green after the sequence.